test/e2e/apps/deployment.go

```diff
@@ -898,19 +907,15 @@ func testProportionalScalingDeployment(f *framework.Framework) {
 	err = framework.WaitForReplicaSetDesiredReplicas(c.ExtensionsV1beta1(), secondRS)
 	Expect(err).NotTo(HaveOccurred())
 
-	numReplicaDiff := newReplicas - replicas
-	if numReplicaDiff < 0 {
-		numReplicaDiff = -numReplicaDiff
-	}
-	biggerProportion := (numReplicaDiff + 1) / 2
-	availableReplicas = availableReplicas + biggerProportion
-	framework.Logf("Verifying that first rollout's replicaset has %d available replicas", availableReplicas)
-	err = framework.WaitForReplicaSetAvailableReplicas(c.ExtensionsV1beta1(), firstRS, availableReplicas)
-	Expect(err).NotTo(HaveOccurred())
+	// First rollout's replicaset should have .spec.replicas = 8 + (30-10)*(8/13) = 8 + 12 = 20 replicas.
```

Comment:
@janetkuo Is this explanation clear enough? Tell me if it can be clearer.

According to documentation: `Any leftovers are added to the ReplicaSet with the most replicas.`
I realized that it is impossible to test against number of leftovers without having more than 2 RSs. For 2 RSs, any additional number of replicas is proportionated to the 2 RSs without any leftover.

In this example, `(30-10)*(8/13) = 12.31` and `(30-10)*(5/13) = 7.69`. By rounding to nearest integer, 12 replicas are given to first rollout's RS, while 8 replicas are given to second rollout's RS. No leftover is possible.

Leftover is possible with 3 RSs. Let's say there are 20 additional replicas, and the 3 RSs have .